Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx

```diff
@@ -1040,6 +1040,8 @@ class TPCTimeSeries : public Task
           const auto& trkOrig = tracksTPC[iTrk];
           const bool isNearestVtx = (idxITSTPC.back() == -1); // is nearest vertex in case no vertex was found
           const float mx_ITS = hasITSTPC ? tracksITSTPC[idxITSTPC.front()].getX() : -1;
+          const float pt_ITS = hasITSTPC ? tracksITSTPC[idxITSTPC.front()].getQ2Pt() : -1;
```

Comment:
OK, but not sure `-1` is a good dummy value for the q2Pt, perhaps an "impossible" value like -999 would be more appropriate.

Reply:
I assume `-1` is fine, since we also store the `hasITSTPC` flag, which we use to select the ITS-TPC tracks.